os/booting-on-launchtimevps.md

```diff
@@ -0,0 +1,133 @@
+<div class="row">
+  <div class="col-lg-8 col-md-10 col-sm-8 col-xs-12 co-m-screenshot">
+    <a href="rimuhosting-coreos-image-select-cloud-config.png">
+      <img src="img/rimuhosting-coreos-image-select-cloud-config.png" />
+    </a>
+    <div class="co-m-screenshot-caption">Choosing the CoreOS image and providing a cloud config.</div>
+  </div>
+</div>
+4.  Start the install.  The server will be setup and you will be notified when that is complete.
+
+A VM can be reinstalled (with a fresh/clean CoreOS image and different cloud-config) via the [reinstall interface][rh-reinstall-page].
+
+### Via the API
+
+Set your public [SSH keys][rh-ssh-keys-page].
+
+Git clone the [Python driver for the RimuHosting API][rh-python-driver-api]
+```sh
+git clone git@github.com:pbkwee/RimuHostingAPI.git
+```
+
+Install the library:
+```
+python3 setup.py build install
+```
+
+If you do not already have a server with us, you will need to email RimuHosting and ask them to enable automated server setups on your account.
+
+Get a [server management API key][rh-api-keys-page]
+
+Set the API key as an environment variable:
+
+```sh
+export RIMUHOSTING_APIKEY=00000000123456789
+```
+
+Or set the key in a `.rimuhosting` config file:
+```sh
+echo "export RIMUHOSTING_APIKEY=00000000123456789" >> ~/.rimuhosting
+```
+
+Edit the server spec you wish to create at `sample-configs/unmodified/servers/server.json` e.g. 
+
+```json
+{
+  "vps_parameters": {
+    "disk_space_mb": "8192",
+    "memory_mb": "4096"
+  },
+  "instantiation_options": {
+    "distro": "coreos.64",
+    "domain_name": "coreos-master.example.com"
+  }
+}
+```
+
+
+Edit the cloud config data you wish to use at `sample-configs/defaults/cloud-init/master.yaml`.
+
+Create the CoreOS VM:
+
+```sh
+$ python3 mkvm.py --server_json sample-configs/unmodified/servers/server.json \
+--cloud_config sample-configs/defaults/cloud-init/master.yaml
+```
+
+For more details, check out [RimuHosting's API documentation][rh-api-docs].
+
+### Adding more machines
+To add more instances to the cluster, just launch more with the same
+cloud-config. New instances will join the cluster regardless of VM location.
+
+## SSH to your VM
+To connect to a VM after it's created (takes a couple of minutes), run:
+
+```sh
+ssh core@<ip address>
+```
+
+Optionally, you may want to [configure your ssh-agent]({{site.baseurl}}/docs/launching-containers/launching/fleet-using-the-client/#remote-fleet-access) to more easily run [fleet commands]({{site.baseurl}}/docs/launching-containers/launching/launching-containers-fleet/).
+
+## Using CoreOS
+
+Now that you have a cluster bootstrapped it is time to play around.
+Check out the [CoreOS Quickstart]({{site.baseurl}}/docs/quickstart) guide or dig into [more specific topics]({{site.baseurl}}/docs).
```

Comment:
Check out the [CoreOS quickstart guide](quickstart.md) or dig into [more specific topics]({{site.baseurl}}/docs/).